R/ExtractData_Soils.R

```diff
@@ -263,6 +263,208 @@ extract_soil_CONUSSOIL <- function(MMC, sim_size, sim_space, dir_ex_soil,
   MMC
 }
 
+#' Extracts soils data (clay, sand, \var{bd}, gravel, depth, ...) from \var{.tif} files in
```

Comment:
This is a function specifically to extract soil data from ISRIC's SoilGrids250m dataset. A useful documentation title does provide such function-specific information. Referencing tif-formatted files or referring to the path `dir_ex_soil` where all our soil datasets are stored doesn't tell the user of this function what it does.